blog/models.py

```
@@ -12,6 +19,56 @@
 
+    def save(self, *args, **kwargs) -> None:
+        # call default impl
+        super().save(*args, **kwargs)
+
+        # hook notifications in
+        self._send_notifications()
+
+    def _send_notifications(self) -> None:
+        try:
+            self._send_emails()
+        except ConnectionError as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)